lib/base/application.cpp

```diff
@@ -34,6 +34,10 @@
 
 using namespace icinga;
 
+#ifdef _WIN32
+#define EXCEPTION_CODE_CXX_EXCEPTION 0xe06d7363
```

Comment:
Where does this value come from? Would you be able to add a comment or even link some sort of documentation?

Reply:
The best documentation I could find is https://devblogs.microsoft.com/oldnewthing/20160915-00/?p=94316, I'm not absolutely sure if it's actually an implementation detail or not, but it seems to be quite stable. I will add this as a comment.

Fun fact: `echo 6d7363 | xxd -r -p` -> "msc"

Reply:
I don't know why Github doesn't show this as code block as outdated in the conversation tab, but there is an updated version.